src/main/java/org/spongepowered/api/world/server/ChunkManager.java

```diff
@@ -246,4 +249,65 @@ default CompletableFuture<Boolean> regenerateChunk(final Vector3i chunkPosition,
      */
     CompletableFuture<Boolean> regenerateChunk(int cx, int cy, int cz, ChunkRegenerateFlag flag);
 
+    /**
+     * Returns the path of the directory where region files are stored.
+     *
+     * @return The path of the region directory
+     */
+    Path regionDirPath();
+
+    /**
+     * Returns the path of the region file containing the given {@link WorldChunk}.
+     *
+     * @param chunk The {@link WorldChunk} for which to acquire the region file path
+     * @return The path of the region file for the given chunk
+     */
+    Path regionFilePath(WorldChunk chunk);
```

Comment:
I'd almost want to have a StorageHandler interface to deal with this so that it's not directly exposed in the API, but can be something exposed for "vanilla" kinds.

Reply:
It's not always guaranteed that the world has region files (like mods) that could be storing chunks in a different region handler that we simply won't be able to directly interface with.